.github/workflows/tests.yaml

```diff
@@ -43,6 +43,9 @@ jobs:
       matrix:
         os: [ubuntu-20.04, windows-latest, macos-latest]
         pyv: ["3.8", "3.9", "3.10"]
+        pytest-filter: 
+        - "import or plot or live or experiment"
```

Comment:
Using `import or plot or live or experiment` to try to split Windows tests in half.